futurex_openedx_extensions/helpers/tenants.py

```diff
@@ -54,9 +54,10 @@ def fix_lms_base(domain_name: str) -> str:
         return ''
 
     lms_root_parts = urlparse(settings.LMS_ROOT_URL)
+
+    if domain_name[:4] != 'http':
+        domain_name = f'{lms_root_parts.scheme}://{domain_name}'
```

Comment:
refactoring the way scheme is added because the original code produces a bug in python 3.11

```shell
$ python3.8 -c "from urllib.parse import urlparse; print(urlparse('example.com:8080'))"
ParseResult(scheme='', netloc='', path='example.com:8080', params='', query='', fragment='')
```

```bash
$ python3.11 -c "from urllib.parse import urlparse; print(urlparse('example.com:8080'))"
ParseResult(scheme='example.com', netloc='', path='8080', params='', query='', fragment='')
```